src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -170,11 +174,11 @@ export class CalcitePickListItem {
         selected={this.isSelected}
       >
         {this.renderIcon()}
-        <div class={CSS.label}>
-          <h4 class={CSS.heading}>{this.textHeading}</h4>
-          <p class={CSS.description}>{this.textDescription}</p>
-        </div>
-        <div onClick={this.secondaryActionContainerClickHandler}>
+        <label class={CSS.label}>
```

Comment:
maybe we should rename the prop from `textHeading` to `textLabel` to be more accurate. This would be a breaking change unless we deprecated but kept backwards compatibility for `textHeading` so I'd suggest we do that.

Reply:
We could do this in a separate issue. Yeah?

Reply:
yea non-blocking. although it could make sense doing it here because this is the PR that's changing the markup and semantic meaning.

Reply:
I do agree tho.

Reply:
Added an issue #288